spec/gameSpec.js

```diff
@@ -0,0 +1,85 @@
+describe("Game", function(){
+
+  var game;
+
+  beforeEach(function(){
+    game = new Game();
+  });
+
+  it("has no intial rolls", function(){
+    expect(game.rolls).toEqual([]);
```

Comment:
It feels like this is testing state, rather than behaviour.  Why is this not ideal? Are there other places that are testing behaviour instead of state?